index.html

```diff
@@ -64,8 +64,31 @@
     <h2>
       Introduction
     </h2>
+    <p>
+      This document defines an API enabling a website to request presentation of a verifiable digital credential (VDC).
+    </p>
+    <p>
+      The API design is agnostic to both credential presentation exchange protocols and credential formats. However, this document includes a registry of credential presentation exchange protocols to ensure interoperability.
+    </p>
+    <p>
+      The API is designed to support the following key properties:
```

Comment:
The word "properties" is too ambiguous.

Reply:
goals?

Reply:
+1 to "goals", seems more common in these types of docs.